src/configuration/quickstart/cifar.yml

```diff
@@ -0,0 +1,47 @@
+trainer:
+  name: lipizzaner_gan
+  n_iterations: 200
+  calculate_net_weights_dist: True
+  # independent_probability, exact_proportion
+  mixture_generator_samples_mode: exact_proportion
+  params:
+    population_size: 1
+    tournament_size: 2 
+    n_replacements: 1
+    default_adam_learning_rate: 0.0003
```

Comment:
Oh, I left it because it was there originally in the quickstart config files. But now, I've copied the files into src/configuration/quickstart-semi-supervised-learning specifically for SSL and I've removed the default_adam_learning_rate there.